templates/battles/battles_list_react.html

```diff
@@ -0,0 +1,24 @@
+{% load render_bundle from webpack_loader %}
```

Comment:
Once every page is implemented with react, these templates should be removed and only the `index` page should contain a django template - the others should only be accessible through the routers.

Reply:
I'll have to figure a way to Django stop giving 404 error whenever someone tries to access the specific URL (just writing down to remember later).